holoviews/operation/datashader.py

```diff
@@ -322,24 +328,24 @@ def get_agg_data(cls, obj, category=None):
         else:
             x, y = dims
 
+        bool_dd = bool(dd)  # Lazy module
         if len(paths) > 1:
             if glyph == 'line':
                 path = paths[0][:1]
-                if lazy_isinstance(path, "dask.dataframe:DataFrame"):
+                if bool_dd and isinstance(path, dd.DataFrame):
```

Comment:
Cannot this be just `if dd and instance(path, dd.DataFrame)`?

Reply:
It is done this way so we don't check `sys.modules` multiple times, if it has been installed but not imported.

Reply:
If it's done this way for performance reasons, maybe write it down, as someone refactoring the code would be likely to clean this up.

Reply:
I updated the text above to clarify this.